### ISSUE TITLE
Since the first version I can not connect with ha-app. Anytime I get error code 1

### ISSUE THREAD

**Issue by abeksis** [estevez-dev/ha_client#228]
`1125 19:08:55 [Debug] :  <!!!> Creating new HomeAssistant instance
1125 19:08:55 [Debug] : AppLifecycleState.inactive
1125 19:08:55 [Debug] : AppLifecycleState.resumed
1125 19:08:55 [Error] : Invalid argument(s): 56.0
1125 19:08:55 [Error] : NoSuchMethodError: The getter ‘scrollOffsetCorrection’ was called on null.
Receiver: null
Tried calling: scrollOffsetCorrection
1125 19:08:55 [Error] : NoSuchMethodError: The getter ‘visible’ was called on null.
Receiver: null
Tried calling: visible
1125 19:08:55 [Debug] : Use lovelace is true
1125 19:08:55 [Debug] : Socket connecting…
1125 19:08:55 [Error] : Invalid argument(s): 56.0
1125 19:08:55 [Error] : NoSuchMethodError: The getter ‘scrollOffsetCorrection’ was called on null.
Receiver: null
Tried calling: scrollOffsetCorrection
1125 19:08:55 [Error] : NoSuchMethodError: The getter ‘visible’ was called on null.
Receiver: null
Tried calling: visible
1125 19:09:10 [Error] : Socket connection timeout
1125 19:09:10 [Error] : Socket stream Error: null
1125 19:09:10 [Debug] : Automatic reconnect is false
1125 19:09:13 [Debug] : Socket sink closed
1125 19:09:55 [Debug] : AppLifecycleState.inactive
1125 19:09:55 [Debug] : AppLifecycleState.paused
1125 19:10:34 [Debug] : AppLifecycleState.inactive
1125 19:10:34 [Debug] : AppLifecycleState.resumed
1125 19:10:34 [Debug] : Use lovelace is true
1125 19:10:34 [Debug] : Socket connecting…`

**Comment by estevez-dev** [comment]
Do you have the same issue if "Use Lovelace UI" is disabled in settings?

**Comment by abeksis** [comment]
Yes, the error message has always been - I tried everything, but the same error.

**Comment by estevez-dev** [comment]
Ok, let's see. If it is possible, I need a screenshot of app when you have this error shown (with error code). Also please tell me what device do you use and what Android version it has. Thanks.

**Comment by abeksis** [comment]
Thanks for trying to help.

I'm using an Android 8 android 8
![screenshot_20181125-195903](https://user-images.githubusercontent.com/10877299/48982550-c9991100-f0ec-11e8-8981-fd868019d8e8.jpg)

**Comment by estevez-dev** [comment]
Ok, now I need a screenshot of your settings. You can hide domain and password if you want.

**Comment by abeksis** [comment]
Do you need anything else? I really want to connect through the app.
![01](https://user-images.githubusercontent.com/10877299/48983217-ab371380-f0f4-11e8-9446-d3b487bdf166.jpg)

**Comment by estevez-dev** [comment]
1. Is there a local IP address in the first field? Or it is some domain?
2. How did you access your Home Assistant web UI? With the same domain/IP address and port?

**Comment by abeksis** [comment]
1. This is a domain. I connect to the same address both through the browser of the phone and the computer.
2. yes. My address is used to connect remotely through the computer and the phone.

**Comment by estevez-dev** [comment]
Did you try to connect from WiFi? Or at home? The same issue?

**Comment by abeksis** [comment]
Yes, unfortunately the same problem. Also via wifi also the cellular network

**Comment by estevez-dev** [comment]
What about port number? Do you use :8123 when connecting through browser?

**Comment by abeksis** [comment]
No, I just write down the address - I've also tried with you to disable the port but it does not allow me - there are limitations to your software.

**Comment by estevez-dev** [comment]
This is it!

If you are using http:// prior your Home Assistant domain in browser - use port 80 in the app and switch off "Use ssl"

If you are using https:// prior domain in browser - use port 443 in app and leave "Use ssl" on.

**Comment by abeksis** [comment]
Thanks for the help - now it works - I flag the "Use Lovelace UI" has an error - probably you do not support this card.
![screenshot_20181125-221201](https://user-images.githubusercontent.com/10877299/48984056-77152000-f0ff-11e8-91e1-3e2f2234727d.jpg)
![screenshot_20181125-221223](https://user-images.githubusercontent.com/10877299/48984057-77152000-f0ff-11e8-8070-26c2a27da3db.jpg)

**Comment by estevez-dev** [comment]
Yes, custom cards is not supported by the app

**Comment by abeksis** [comment]
Will it support the future?

**Comment by estevez-dev** [comment]
Maybe I will add some of the most popular custom cards, but there is no chance to support all of them.
